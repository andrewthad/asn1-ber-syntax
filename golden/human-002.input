300B0C04 4164616D A1030201 32
