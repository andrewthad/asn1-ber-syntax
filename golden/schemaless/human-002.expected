Value
    { tagClass = Universal
    , tagNumber = 16
    , contents = Constructed
        ( fromListN 2
            [ Value
                { tagClass = Universal
                , tagNumber = 12
                , contents = Primitive
                    [ 0x41
                    , 0x64
                    , 0x61
                    , 0x6d
                    ]
                }
            , Value
                { tagClass = ContextSpecific
                , tagNumber = 1
                , contents = Constructed
                    ( fromListN 1
                        [ Value
                            { tagClass = Universal
                            , tagNumber = 2
                            , contents = Primitive [ 0x32 ]
                            }
                        ]
                    )
                }
            ]
        )
    }
